Formula/box2d.rb

```diff
@@ -16,6 +28,7 @@ class Box2d < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "doctest" => :test
```

Comment:
Don't you need this at build-time?

Reply:
No, only at test time.  I had it as a build dependency when I was symlinking but now that's irrelevant.